test/e2e/framework/kubelet_stats.go

```diff
@@ -47,6 +47,7 @@ import (
 
 // KubeletLatencyMetric stores metrics scraped from the kubelet server's /metric endpoint.
 // TODO: Get some more structure around the metrics and this type
+// TODO(alejandrox1): this is already present in test/e2e/framework/metrics.
```

Comment:
i'm assuming these will come in a separate PR?

i don't know about the others but i'm in favor of having multiple commits per PR.
this reduces my notifications spam and is still easy to review, because one can review single commit at a time.

Reply:
Yeah, but this PR is already big enough, when you add the TODOs please open an issue.

Reply:
created https://github.com/kubernetes/kubernetes/issues/79907